modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectRelationship.tsx

```diff
@@ -57,7 +58,9 @@ export default function SelectRelationship({
 			error={error}
 			label={Liferay.Language.get('parameter')}
 			onChange={(event) => {
-				onChange?.(fields[Number(event.target.value)].id);
+				onChange?.(
+					fields.find(({name}) => name === event.target.value)?.id!
+				);
```

Comment:
Hey @naraandrade, could we pass `onChange` as a required prop? since `Select` only works if `onChange` is passed? That way we can remove that null validation (`.?`) on line `61`. Another thing is, I think we can extract the `value` directly from the `event` on line `60`, like:

```tsx
onChange={({target: {value}}) => {
    onChange(
	fields.find(({name}) => name === value)?.id!
    );
}}
```